go/ql/src/change-notes/2013-05-02-post-unicode-normalization-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* Added a new query, `go/post-unicode-normalization`, to detect a misuse of a post-unicode normalization.
```

Comment:
Experimental queries don't need change notes

Reply:
Would you believe me if I tell you that I thought I did that.